P/PhreeqcRM/build_tarballs.jl

```diff
@@ -0,0 +1,90 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "PhreeqcRM"
+version = v"3.8.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/usgs-coupled/phreeqcrm.git", "eb70b492dbdc7818c62b4db39b9f589a2c5c0aed")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+
+# so we can use a newer version of cmake
+apk del cmake
+
+cd $WORKSPACE/srcdir/phreeqcrm
+
+# Configure
+cmake -B Release \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DBUILD_SHARED_LIBS=ON \
+    -DPHREEQCRM_BUILD_MPI=OFF \
+    -DPHREEQCRM_USE_ZLIB=OFF \
+    -DPHREEQCRM_DISABLE_OPENMP=OFF
+
+# Compile
+cmake --build Release --parallel ${nproc}
+
+# Compile tests
+cd Release/Tests
+make all
+cd ../..
+
+# Deploy library
+install -Dvm 755 "Release/libPhreeqcRM.${dlext}" -t "${libdir}"
+
+# Deploy test executables
+install -Dvm 755 "Release/libPhreeqcRM.${dlext}" -t "${libdir}"
+install -Dvm 755 "Release/Tests/TestBMIdtor${exeext}" "${bindir}/TestBMIdtor${exeext}"
+install -Dvm 755 "Release/Tests/TestRM${exeext}" "${bindir}/TestRM${exeext}"
+install -Dvm 755 "Release/Tests/TestRMdtor${exeext}" "${bindir}/TestRMdtor${exeext}"
```

Comment:
This can also be simplified
```suggestion
install -Dvm 755 "Release/Tests/TestBMIdtor${exeext}" -t "${bindir}"
install -Dvm 755 "Release/Tests/TestRM${exeext}" -t "${bindir}"
install -Dvm 755 "Release/Tests/TestRMdtor${exeext}" -t "${bindir}"
```